src/lib.rs

```diff
@@ -28,6 +28,10 @@ pub use api::*;
 #[cfg(any(feature = "telegram-trait", feature = "async-telegram-trait"))]
 pub use api_traits::*;
 
+#[doc(hidden)]
+#[cfg(any(feature = "async-http-client"))]
```

Comment:
The any can be removed here as its only a single requirement here.

```suggestion
#[cfg(feature = "async-http-client")]
```